src/pages/SearchPage.tsx

```
@@ -51,61 +66,40 @@ const SearchPage: FC = () => {
                         }
                     </>
                 }
                 {awaitingResponse && !errorResponse &&
-                        <div className = { 'loader' }>
-                            Loading...
+                    <>
+                        Searching...
+                        <div className = { 'loader_container' }>
+
+                            <img className = { 'loader' } src = { 'loader.png' } alt = { 'Loading...' }>
+                            </img>
```

Biome

[error] img is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)